src/components/History/Scrap/ScrapContent.js

```diff
@@ -48,12 +30,10 @@ const ScrapContent = ({isToggleOpen}) => {
   const [memoItemList, setMemo] = useState([]);
   const [memoIdList, setIdList] = useState([]);
   const [scrapItemList, setScrap] = useState([]);
-  const [pageState, setState] = useState('READ');
   const [currentMail, setCurr] = useState(null);
   const [checkedList, setCheckedList] = useState([]);
```

Comment:
```suggestion
  const [currentMail, setCurrentMail] = useState(null);
  const [checkedList, setCheckedList] = useState([]);
```
명확히 쓰면 좋지 않을까 생각합니다.